### PR TITLE
Added dvclive to tests

### PR DESCRIPTION
**Pull request by daavoo**
* [X] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

Close https://github.com/iterative/dvclive/issues/149 
Close https://github.com/iterative/dvclive/issues/32

### CODE REVIEW

**Comment by daavoo** [comment]
Before:

```
=========================== short test summary info ============================
SKIPPED [11] tests/conftest.py:102: gdrive tests not enabled through CLI
SKIPPED [1] tests/func/test_add.py:711: Windows specific
SKIPPED [11] tests/func/test_live.py:60: no dvclive
SKIPPED [2] tests/func/test_live.py:170: no dvclive
SKIPPED [1] tests/func/test_system.py:8: Windows specific
SKIPPED [1] tests/unit/scm/test_git.py:302: Skipped
SKIPPED [2] tests/unit/scm/test_git.py:332: Skipped
SKIPPED [2] tests/unit/scm/test_git.py:350: Skipped
SKIPPED [1] tests/unit/utils/test_fs.py:110: Windows specific
SKIPPED [1] tests/unit/utils/test_fs.py:226: Windows specific
SKIPPED [1] tests/unit/utils/test_utils.py:116: Windows specific
SKIPPED [1] tests/unit/scm/test_git.py:379: Skipped
SKIPPED [2] tests/unit/scm/test_git.py:405: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:428: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:468: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:492: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:533: Skipped
SKIPPED [4] tests/unit/scm/test_git.py:563: Skipped
SKIPPED [3] tests/unit/scm/test_git.py:585: Windows only
= 2437 passed, 48 skipped, 2 xfailed, 2 xpassed, 36 warnings in 542.65s (0:09:02) =
```

After:

```
=========================== short test summary info ============================
SKIPPED [11] tests/conftest.py:102: gdrive tests not enabled through CLI
SKIPPED [1] tests/func/test_add.py:711: Windows specific
SKIPPED [1] tests/func/test_system.py:8: Windows specific
SKIPPED [1] tests/unit/utils/test_fs.py:110: Windows specific
SKIPPED [1] tests/unit/utils/test_fs.py:226: Windows specific
SKIPPED [1] tests/unit/utils/test_utils.py:116: Windows specific
SKIPPED [1] tests/unit/scm/test_git.py:302: Skipped
SKIPPED [2] tests/unit/scm/test_git.py:332: Skipped
SKIPPED [2] tests/unit/scm/test_git.py:350: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:379: Skipped
SKIPPED [2] tests/unit/scm/test_git.py:405: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:428: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:468: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:492: Skipped
SKIPPED [1] tests/unit/scm/test_git.py:533: Skipped
SKIPPED [4] tests/unit/scm/test_git.py:563: Skipped
SKIPPED [3] tests/unit/scm/test_git.py:585: Windows only
= 2450 passed, 35 skipped, 1 xfailed, 3 xpassed, 36 warnings in 497.45s (0:08:17) =
```

File: setup.py [hunk 1]
```diff
@@ -115,10 +115,13 @@ def run(self):
 all_remotes = gs + s3 + azure + ssh + oss + gdrive + hdfs + webhdfs + webdav
 
 terraform = ssh + ["python-terraform>=0.10.1", "jinja2>=2.0.0"]
+dvclive = ["dvclive>=0.2.1"]
```
Comment (efiop) [inline setup.py:118]: Is there a reason to not add it directly into `test_requirements.txt`?

Comment (daavoo) [reply setup.py:118]: Not really. I just wasn't sure if `test_requirements.txt` was reserved to "strict" test requirements given that `dvclive` depdency is optional and tests would be skipped

Comment (efiop) [reply setup.py:118]: Let's put it into `test_requirements.txt` directly then.

**Review by efiop — APPROVED** [review]

**Pull request merged**